lldb/test/API/functionalities/gdb_remote_client/TestGDBServerTargetXML.py

```diff
@@ -692,6 +692,9 @@ class MyResponder(MockGDBServerResponder):
                 "0102030405060708"  # t4
                 "0102030405060708"  # t5
                 "0102030405060708"  # t6
+                "6162636465666768"  # pc
```

Comment:
I guess that we just didn't provide data for the PC before? Surprised that the test worked at all.